lib/fizzy/stack.hpp

```diff
@@ -61,37 +67,63 @@ class OperandStack
     /// in the constructor after the m_storage.
     Value* m_top;
 
+    Value* m_bottom;
+
+    Value* m_locals;
```

Comment:
Would the order of these three manifest speed differences?

Reply:
I have corrected this. New version may be slightly faster, but hard to tell for sure as they are very close.